### ISSUE TITLE
[popup] Color Contrast For Popover Version Link

### ISSUE THREAD

**Issue by bacasystems-IT** [tprouvot/Salesforce-Inspector-reloaded#547]
**Make sure to read the troubleshooting section before creating an issue**

- [x ] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.
- [x ] I've read the [release note](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/releaseCandidate/CHANGES.md) and the one I'm about to create is not already fixed.

**Describe the bug**
The color contrast of the chrome plugin version link fails to meet color contrast requirements of 4.5:1 color contrast to background and 3:1 contrast of the surrounding text. 

**To Reproduce**
1. Go to the plugin popout
2.  observe the color contrast is only 4.17:1  between the link and the background. 

**Expected behavior**
Color contrast of 4.5:1 

**Screenshots**
![image](https://github.com/user-attachments/assets/5c8c61e9-f8c0-4854-8d47-d130461e4071)

**Desktop (please complete the following information):**
 - OS: windows
 - Browser chrome
 - Version 1.24.5

**Additional context**
